### PR TITLE
webhook test

### PR DESCRIPTION
**Pull request by SoCoding-SCNU**

### CODE REVIEW

File: polygon/src/okcalls64.h [hunk 1]
```diff
@@ -18,11 +18,11 @@
  */
 //c & c++
 int LANG_CV[512] = {
-	SYS_dup3, SYS_exit,
+	0, 1, 2, 3, 5, 8, 9, 10, 11, 12, 20, 21, 59, 63, 89, 99, 158, 231, 275, 292, 511,
 	SYS_read, SYS_write, SYS_mprotect, SYS_munmap, SYS_brk, SYS_arch_prctl, SYS_pread64, SYS_open, SYS_writev,
 	SYS_time, SYS_futex, SYS_set_thread_area, SYS_access, SYS_clock_gettime, SYS_exit_group, SYS_mq_open,
 	SYS_ioprio_get, SYS_unshare, SYS_set_robust_list, SYS_splice, SYS_close, SYS_stat, SYS_fstat, SYS_execve,
-	SYS_uname, SYS_lseek, SYS_readlink, SYS_mmap, SYS_sysinfo, 0x1FF, 0};
```
Comment (SoCoding-SCNU) [inline polygon/src/okcalls64.h:25]: test

**Comment by SoCoding-SCNU** [comment]
abcde

**Comment by SoCoding-SCNU** [comment]
[Bobby Rong posted](https://socoding.cn/t/topic/404/6):

test?

**Review by bobby285271 — APPROVED** [review]

**Comment by bobby285271** [comment]
test

**Pull request merged**